views/includes/task-list/nature-of-business.html

```diff
@@ -4,7 +4,7 @@ <h2 class="app-task-list__section">
 <ul class="app-task-list__items">
     <li class="app-task-list__item">
         <span class="app-task-list__task-name">
-            <a href="/confirmation-statement/sic">
+            <a href="/confirmation-statement/company/{{company.companyNumber}}/sic">
```

Comment:
could you build the url in the controller and pass it into the template as one of the templateOptions? this would make it easier to change the urls in future if we needed to as then we could just change the constants in pageUrls and we wouldn't have to change the urls in the templates.

Reply:
Matt has done it with the PSC link, have a look at this for an example - https://github.com/companieshouse/confirmation-statement-web/blob/9a10cdfa92e50e9e75b0b71e3bb2336cac93ece3/src/types/task.list.ts#L60

Reply:
then you can reference it in the template like this - https://github.com/companieshouse/confirmation-statement-web/blob/main/views/includes/task-list/people.html#L21